definitions/private-law/json/CaseEvent/CaseEvent.json

```diff
@@ -1986,6 +1986,21 @@
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/update-task-list/submitted",
     "EndButtonLabel": "Save and continue",
     "EventEnablingCondition": "caseTypeOfApplication=\"C100\" AND taskListVersion=\"v3\""
+  },
+  {
+    "LiveFrom": "08/05/2024",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "citizenPcqUpdate",
+    "Name": "Update pcq for citizen",
+    "Description": "Event to update pcq id of citizen in case data",
+    "DisplayOrder": 143,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowSummary": "N",
+    "ShowEventNotes": "N",
+    "EndButtonLabel": "Save and continue",
+    "EventEnablingCondition": "caseTypeOfApplication=\"DO_NOT_SHOW\""
```

Comment:
move this to a new folder and a file please. this file is overloaded.
Create a folder called pcq and move it over there please.

Reply:
Lets change the name to PCQ update for Citizen, description is fine.